content/techniques/configuration.md

```diff
@@ -44,6 +44,8 @@ DATABASE_USER=test
 DATABASE_PASSWORD=test
 ```
 
+> info **Notice** For a practical example with asynchronous database configuration see the documentation [here](https://docs.nestjs.com/techniques/database#async-configuration)
```

Comment:
```suggestion
> info **Notice** For a practical example with asynchronous database configuration see the documentation [here](./techniques/database#async-configuration)
```

just in case of `docs.nestjs.com` went down or something, using relative paths will let users explore the markdown file as well